### DISCUSSION TITLE
Integration XBOX Game Bar

### DISCUSSION THREAD

**Discussion by ghost** [Leo-Corporation/Gavilya#89] [Ideas]
Est il envisageable de créer un widget du frontend pour la Game Bar  de Windows ?
L'endroit est parfait, super accessible, en OSD... Gavilya serait le premier !
Plus besoin du shell windows. Nos PC auraient enfin une interface comme les consoles, digne de ce nom :)

**Comment by lpeyr** [comment]
@nolanmars Bonne idée, un widget Gavilya dans la Game Bar pourrait être intéressant, cela pourrait offrir aux utilisateurs un moyen plus rapide de lancer leurs jeux, voire d'obtenir des informations que propose déjà l'application de bureau Gavilya (avis, plateformes...).

Je me suis renseigné sur la création de widget Xbox Game Bar, et en réalité, c'est juste une application UWP, avec le SDK Xbox Game Bar sous forme de package NuGet, donc au niveau du code, pas trop de problème pour moi. Cependant, pour rendre disponible à tous le widget, il faut le publier dans le Microsoft Store, ce que je n'ai jamais fait, mais là aussi, ça n'a pas l'air très compliqué.

Au niveau de la distribution du widget, cela se fera à côté de Gavilya, c'est-à-dire que le widget ne sera pas synchronisé avec les versions stables de Gavilya.

Je n'ai pas encore décidé quand je m'attaquerai au widget, donc patience :smiley: !

> _Note : Il est possible que cette fonctionnalité ne soit jamais intégrée à Gavilya._

**Comment by ghost** [comment]
Pas de souci, ce n'est qu'une suggestion ;)
A priori on peut installer directement les widgets pour tester sans passer par le store (certains -trés peu- widgets amateurs sont sur GitHub).

Effectivement obtenir des informations en rapport avec le jeux serait un plus, très bonne idée :D

En tous cas si besoin de test, i'am here !
